TS06-Facets-And-NDVI-Plotting-In-R.Rmd

```diff
@@ -1,14 +1,15 @@
 ---
 layout: post
-title: "Lesson 06: Plot using Facets and Plot Time Series with NDVI data"
+title: "Lesson 06: Create Plots with Multiple Panels, Grouped by Time Using GGPLOT
```

Comment:
Same comment. It stands out in a title well and will grab someones attention visually. let's discuss.
I think it's ok if we are consistent with use. I think people will see "ggplot" and want to view the lesson.